cluster/cluster.yaml

```diff
@@ -104,15 +163,27 @@ Resources:
           Value: owned
       VpcId: "{{.Cluster.ConfigItems.vpc_id}}"
     Type: 'AWS::EC2::SecurityGroup'
-  MasterLoadBalancerVersionDomain:
+{{- end }}
+  MasterLoadBalancerDNSRecord:
```

Comment:
What actually happens when you rename a CF resource? Does it leave the physical resource untouched, or is there a moment where it won't exist?

Reply:
Not sure, I'll try it out.

Reply:
In this case it doesn't work. It tries to create the new resource first (including the DNS name) and that fails because the DNS name already exists. I changed it back to use the original name.